catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,47 +82,71 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const AlbumRow = styled.ul`
+    display: flex;
+    flex-flow: row no-wrap;
```

Comment:
Are both of these be the initial values? Does everything stay the same without this line?

Reply:
good point, I took the values that the library added and ensured it was there in the custom component